tdvt/tdvt/tdvt.py

```diff
@@ -400,41 +403,68 @@ def usage_text():
     ie cast_calcs.*.tds for cast_calcs.sqlserver.tds etc.
 
     Run one logical query test against many datasources
-        py -3 -m tdvt.tdvt --logp logicaltests/setup/calcs/setup.BUGS.B1713.?.xml --tdp cast_calcs.*.tds --run postgres
+        run-pattern --logp logicaltests/setup/calcs/setup.BUGS.B1713.?.xml --tdp cast_calcs.*.tds postgres
 
     This can be combined with * to run an arbitrary set of 'correct' logical query tests against a datasources
-        py -3 -m tdvt.tdvt --logp logicaltests/setup/calcs/setup.BUGS.*.?.xml --tdp cast_calcs.*.tds --run postgres
+        run-pattern --logp logicaltests/setup/calcs/setup.BUGS.*.?.xml --tdp cast_calcs.*.tds  postgres
```

Comment:
Not sure if the two spaces between `cast_calcs.*.tds` and `postgres` are intentional, but if not: 
```suggestion
        run-pattern --logp logicaltests/setup/calcs/setup.BUGS.*.?.xml --tdp cast_calcs.*.tds postgres
```